homeassistant/components/yeelight/light.py

```diff
@@ -582,6 +582,7 @@ def extra_state_attributes(self):
         attributes = {
             "flowing": self.device.is_color_flow_enabled,
             "music_mode": self._bulb.music_mode,
+            "music_mode_state": self._bulb.music_mode_state,
```

Comment:
Why is this added as a state attribute?

Reply:
It was added for debugging purposes. In the new version of the library, music_mode tracks intended state while music_mode_state tracks the actual connection type.

Reply:
> It was added for debugging purposes.

I don't think we should add it to the state machine (and thus memory) and record it in our database/state history because of that.

Reply:
Please use logging for debugging instead.  We don't want to store anything more in the state machine or database than is absolutely needed.

Reply:
I will remove that then. I had only added it because I noticed music mode was there as well.